src/blog/2024-09-14/index.md

```diff
@@ -0,0 +1,73 @@
+title: Pre-Alpha v8.0 Release Note
+summary: Release note for Neotokyo; Rebuild Pre-Alpha v8.0
+author: nullsystem
+
+# Pre-Alpha v8.0 Release Note
+2024-09-14 nullsystem
+
+* [Builds download + changelog](https://github.com/NeotokyoRebuild/neo/releases/tag/v8.0-prealpha)
+
+## Highlights
+
+### Weapon spread and recoil
+Spread and recoil changes are now in-place and is parity/close to to OG:NT. There is now a difference
+between hip fire and ADS, giving a more tighter spread for ADS.
+
+### New main menu
+The main menu got an entire overhaul replacing the standard Source main menu with a custom replacement. The settings,
+server browser, new games, popups, and player lists got replaced. The work on the UI is currently on-going and while
+the server browser is missing some extras, it is usable enough to replace the old UIs currently. However, the "legacy"
+windows can be accessed in the server browser and settings by the "legacy" button on the bottom.
+
+Here's a screenshot of the settings page:
+
+[![Options menu with the new overhaul GUI](neoimgui.png_thumb.png)](neoimgui.png)
+
+For the rest, load up NT;RE Pre-Alpha v8.0 and see for yourselves.
+
+## New features
+
+### Old Squad HUD
+You can now switch to the OG:NT-style squad/round HUD by using the convar: `neo_cl_squad_hud_original = 1`. The 
+squad stars scaling can be tweaking using `neo_cl_squad_hud_star_scale`, using `1` to scale from 1080p.
+
+### Mirror team damage
+Mirror team damage feature is now in with multiple server-side convars to tweak it. By default this is turned on with
```

Comment:
```suggestion
Mirror team damage server commands are now available, and can be tweak at the server operator's discretion. By default this is enabled with
```

Reply:
I found it a bit weird to call it server commands rather than feature even if it's tweaked as one.